src/Normalizer/AttributeOptionNormalizer.php

```diff
@@ -2,29 +2,44 @@
 
 namespace Flagbit\Bundle\TableAttributeBundle\Normalizer;
 
-use Pim\Bundle\EnrichBundle\Normalizer\AttributeOptionNormalizer as BaseNormalizer;
-
-/**
- * Attribute option normalizer for internal api
- *
- * @package Flagbit\Bundle\TableAttributeBundle\Normalizer
- * @author Ruben Beglaryan <ruben.beglaryan@flagbit.de>
```

Comment:
If you remove the `@author` and `@package` annotations, please do it consistently thought he whole project.